GitUI/ScriptsEngine/IScriptHostControl.cs

```diff
@@ -7,5 +7,8 @@ public interface IScriptHostControl
         GitRevision? GetLatestSelectedRevision();
         IReadOnlyList<GitRevision> GetSelectedRevisions();
         Point GetQuickItemSelectorLocation();
+
+        void GridRefresh();
+        void GoToRef(string? refName, bool showNoRevisionMsg, bool toggleSelection = false);
```

Comment:
I named this to match with `needsGridRefresh`, and I assume the reason for the change is `VerbNoun`. Like above, it would help if there was an explanation.

Reply:
Actually, no, as `NounVerb` it would be `RefreshGrid`, not `RefreshRevisionGrid`. So I still need an explanation for this one.

Reply:
Yes it is `VerbNoun` and the noun *is* `RevisionGrid`. Otherwise, "Which grid"?

Reply:
Though rather `RefreshRevisions` because it does not matter how they are displayed.
Or even `Refresh` because in the future there might be more to update.

Reply:
I like `Refresh`. I'm using it.